research-software-directory/RSD-as-a-service
# 811 fix badges

# Pull request Title

Fixes #811 

Changes proposed in this pull request:

*    Update the readme to point to the correct test badges

How to test:

*    Check the the badge URL in the README are valid

PR Checklist:
